pinot-connectors/pinot-spark-connector/pom.xml

```diff
@@ -36,7 +36,7 @@
         <spark.version>2.4.5</spark.version>
         <circe.version>0.13.0</circe.version>
         <paranameter.version>2.8</paranameter.version>
-        <jackson.module.scala.version>2.9.8</jackson.module.scala.version>
+        <jackson.module.scala.version>2.10.0</jackson.module.scala.version>
```

Comment:
Could you add info in the PR description on why all these pom file changes are needed?

Reply:
updated